site/src/components/components/LivePreview.tsx

```diff
@@ -122,9 +215,12 @@ export const LivePreview: FC<LivePreviewProps> = ({
         </div>
 
         {showCode && (
-          <Pre className={styles.codePreview}>
-            <div className="language-tsx">{ComponentExample.sourceCode}</div>
-          </Pre>
+          <>
+            <Pre className={styles.codePreview}>
+              <div className="language-tsx">{ComponentExample.sourceCode}</div>
+            </Pre>
+            <Button onClick={editInStackBlitz}>Edit in StackBlitz</Button>
```

Comment:
You could align it right, and add a tear out icon.
Maybe also make it look like it is over the codeblock by adding background colour.